### PR TITLE
Add missing keyboard symbols

### LINKED ISSUE #106: Showing an action that uses certain symbols will crash the game
In my game, players can press `,` (Comma) to perform a certain action. Trying to show this as a key hint using this plugin simply results in the game crashing.

Here's a list of symbols I've been able to replicate this crash with:
- `,` Comma
- `.` Period
- `/` Forward Slash
- `'` Apostrophe
- `=` Equals
- \` backtick

Using any of these symbols results in the following error:
```
E 0:00:00:0454   ControllerIconTexture.gd:367 @ _stitch_texture(): Invalid image: null
  <C++ Error>    Condition "p_image.is_null()" is true. Returning: Ref<ImageTexture>()
  <C++ Source>   scene/resources/image_texture.cpp:76 @ create_from_image()
  <Stack Trace>  ControllerIconTexture.gd:367 @ _stitch_texture()
                 ControllerIconTexture.gd:379 @ _get_rid()
```

Reading the error message, it just looks like the icons are missing from the keyboard icon set. I can probably learn how the icons were made and contribute some icons to the pack, if necessary.

Interestingly, this error happens even if the *shown* key is not one of these symbols. For example, I use a Dvorak keyboard, and where `,` is on QWERTY, it should show `W`. However, it instead throws the error above. It does display the correct key if I use other keys, though, so I'm not sure what could cause this since it shouldn't be looking for `,` when displaying `W`.

### PR DESCRIPTION
**Pull request by TacticalLaptopBag**
Fixes #106

Adds additional missing keyboard symbols and maps `QuoteLeft` to the icon for `tilde` as the tilde key `~` is more recognizable than backtick (`), and the mapping was missing despite the icon being present.

Symbols added:
- Apostrophe `'`
- Comma `,`
- Period `.`
- Equals `=`
- Forward slash `/`

Each of the icons were created by modifying other existing icons, so the overall style is retained.

### CODE REVIEW

**Comment by TacticalLaptopBag** [comment]
Possible problem: For some reason, on my laptop, assigning an action to `,` will make controller icons look for `KEY_KP_PERIOD`, despite it being a comma and not a period. This could be a weird Dvorak problem though, I will continue to investigate

**Comment by TacticalLaptopBag** [comment]
image

I understand the root of the problem now. In `_convert_event_to_path` it checks if the keycode is 0. If it is, it gets the keycode from the DisplayServer using the physical keycode. For whatever reason, the DisplayServer believes that `,` comma is actually KEY_KP_PERIOD. I need to test whether this is just my laptop, Dvorak doing weird things, or if this is actually a problem.

**Comment by TacticalLaptopBag** [comment]
This is a Dvorak issue. I have added `KEY_KP_PERIOD` to the key match since that should map to period anyway. This problem isn't an issue with Controller Icons, so this PR is ready once again

File: addons/controller_icons/ControllerIcons.gd [hunk 1]
```diff
@@ -574,10 +574,20 @@ func _convert_key_to_path(scancode: int):
 			return "key/bracket_left"
 		KEY_BACKSLASH:
 			return "key/slash"
+		KEY_SLASH:
+			return "key/forwardslash"
```
Comment (rsubtil) [inline addons/controller_icons/ControllerIcons.gd:578]: For consistency with other asset names, please add an underscore to separate these words (rename the icon as well):
```suggestion
			return "key/forward_slash"
```

File: addons/controller_icons/ControllerIcons.gd [hunk 2]
```diff
@@ -574,10 +574,20 @@ func _convert_key_to_path(scancode: int):
...
 		KEY_BRACKETRIGHT:
 			return "key/bracket_right"
-		KEY_ASCIITILDE:
+		KEY_ASCIITILDE, KEY_QUOTELEFT:
```
Comment (rsubtil) [inline addons/controller_icons/ControllerIcons.gd:581]: `KEY_QUOTELEFT` should be mapped to a dedicated \` character instead. Could you create an icon specific to it as well?

**Review by rsubtil — CHANGES_REQUESTED** [review]
Thank you for your contribution, and for creating new icons not available in the original pack! There are two small issues that need to be agdressed, but other than that I'm happy to merge this.

Additionally, the new icons you created are derivative works, and thus are not automatically licensed with the same original license (CC0). I can only accept these if you choose to license them with the same license ([description of CC0](https://creativecommons.org/publicdomain/zero/1.0/)). Please confirm if you agree with this. I will still credit you for the creation of these assets on the main README file of this repository.

**Comment by TacticalLaptopBag** [comment]
I agree to using the license - I'll get to work on the changes as soon as I can

**Comment by TacticalLaptopBag** [comment]
Created an icon for `KEY_QUOTELEFT` and renamed forwardslash -> forward_slash

**Review by rsubtil — APPROVED** [review]

**Comment by rsubtil** [comment]
Thank you so much for your contribution!

**Pull request merged**